README.md

```diff
@@ -22,7 +22,7 @@ Readme](SECURITY.md).
 Download
 --------
 
-Download [the JAR][]. Or for Maven, add to your `pom.xml`:
+Download [the JARs][]. Or for Maven, add to your `pom.xml`:
```

Comment:
I wasn't yet trying to dissuade people from using grpc-all. It just is that downloading grpc-all is pretty useless. Using it in maven or gradle still has the same effect as it used to.

I think avoiding grpc-all will need a reworking of the docs.